client/question/question.js

```diff
@@ -19,6 +19,16 @@
 window.tie = angular.module('tie', ['ui.codemirror', 'tieConfig', 'tieData']);
 
 /**
+ * Setting app to html5mode so that we don't have to have #'s in the url.
```

Comment:
Shouldn't this go in the global tie config for all pages?

Reply:
Mm not sure why, but when I tried it in the config file, it wouldn't route correctly to the questions.

Reply:
I'm silly, that's why. Moved.